SslStress fails to build on Linux

```
/app/System.Net.Security/tests/StressTests/SslStress/Directory.Build.targets(2,3): error MSB4019: The imported project "/live-runtime-artifacts/targetingpacks.targets" was not found. Confirm that the expression in the Import declaration "/live-runtime-artifacts/targetingpacks.targets" is correct, and that the file exists on disk. [/app/System.Net.Security/tests/StressTests/SslStress/SslStress.csproj]

Build FAILED.
```

https://dev.azure.com/dnceng/public/_build/results?buildId=1886274&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=6c56d998-158d-5232-283c-5140104799fc

It also repros locally for me. `/live-runtime-artifacts` is missing from the SslStress container image filesystem, possible explanation is that `run-docker-compose.sh` fails to propagate the correct correct `$SDK_BASE_IMAGE`.

--- a/src/libraries/System.Net.Security/tests/StressTests/SslStress/run-docker-compose.sh
+++ b/src/libraries/System.Net.Security/tests/StressTests/SslStress/run-docker-compose.sh
@@ -71,7 +71,7 @@ if [[ "$buildcurrentlibraries" -eq 1 ]]; then
 fi
 
 build_args=""
-if [[ -z "$imagename" ]]; then
+if [[ -n "$imagename" ]]; then
     build_args=" --build-arg SDK_BASE_IMAGE=$imagename"
 fi
 

